frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -627,14 +627,12 @@ describe("scenarios > collection_defaults", () => {
         it("should be possible to bulk archive items (metabase#16496)", () => {
           cy.visit("/collection/root");
           selectItemUsingCheckbox("Orders");
-          selectItemUsingCheckbox("Orders in a dashboard", "dashboard");
```

Comment:
Adding one more item doesn't increase our confidence about this functionality, but it increases the length of the test run. Now, selecting ALL items and making sure this functionality still works **could be** an edge case worth checking.
But let's follow up with the separate PR. I don't want to butcher your great fix with these changes anymore :)